lib/LedgerSMB/Template/XLS.pm

```diff
@@ -149,13 +149,11 @@ sub _xls_process {
             cell => \&_cell_handler,
             formula => \&_formula_handler,
             format => \&_format_handler,
-=pod
-            bold => sub { &_named_format('bold', @_) },
-            hidden => sub { &_named_format('hidden', @_) },
-            italic => sub { &_named_format('italic', @_) },
-            shadow => sub { &_named_format('shadow', @_) },
-            strikeout => sub { &_named_format('strikeout', @_) },
-=cut
+#            bold => sub { &_named_format('bold', @_) },
+#            hidden => sub { &_named_format('hidden', @_) },
+#            italic => sub { &_named_format('italic', @_) },
+#            shadow => sub { &_named_format('shadow', @_) },
+#            strikeout => sub { &_named_format('strikeout', @_) },
             },
```

Comment:
Is it vital to retain the comment block? If so, maybe add a comment as to why? In general, I'm a big believer of thorough clean-up